app/styles/play/campaign-view.sass

```diff
@@ -376,7 +376,7 @@ $gameControlMargin: 30px
         height: 32px
         color: white
 
-      .gem, .player-level-icon, .player-hero-icon
+      .gem, .level-indicator .player-level-icon, .player-hero-icon
```

Comment:
Thanks for catching this, I added a new pull request that cleans this up. This part actually shouldn't be there at all, and the typo prevented me from seeing it break the positioning.